octopus_deployment.ps1

```diff
@@ -0,0 +1,15 @@
+$octoYamlPath = [System.IO.Path]::GetFullPath((Join-Path ${env:GITHUB_WORKSPACE} ".octopus/workflow/octopus.yaml"))
+if (Test-Path -Path $octoYamlPath -PathType Leaf)
+{
+    $octoDeploymentSteps = yq eval -j $octoYamlPath
```

Comment:
Can you try with `ce yq` since PowerShell does not throw errors for commands outside PowerShell.